src/components/lava-coder/lava-constants.ts

```diff
@@ -6,6 +9,13 @@ export const defaultResidual = 5;
 export const defaultVentLatitude = 19.5;
 export const defaultVentLongitude = -155.565;
 
+export const minEruptionVolume = 1000000;
+export const maxEruptionVolume = 10000000000;
+
+export const minResidual = 2;
+export const maxResidual = 50;
```

Comment:
Really? 50 seems excessive, but what do I know. 🤷‍♂️

Reply:
Agreed, but that's what's specified in the Jira stories. I suspect that once the researchers get a chance to play around a bit they'll want to make a few changes.